src/System.Threading.Overlapped/tests/ThreadPoolBoundHandle_Helpers.cs

```diff
@@ -19,14 +20,14 @@ struct NonBlittableType
         public string s;
     }
 
-    private static ThreadPoolBoundHandle CreateThreadPoolBoundHandle()
+    private ThreadPoolBoundHandle CreateThreadPoolBoundHandle()
     {
         return CreateThreadPoolBoundHandle((SafeHandle)null);
     }
 
-    private static ThreadPoolBoundHandle CreateThreadPoolBoundHandle(SafeHandle handle)
+    private ThreadPoolBoundHandle CreateThreadPoolBoundHandle(SafeHandle handle)
```

Comment:
@stephentoub I had to remove the `static` to these two methods, because `GetTestFilePath()` is not static. Any reason why we decided to not have them be static?